peer/brontide.go

```diff
@@ -450,13 +452,18 @@ type Brontide struct {
 	queueQuit chan struct{}
 	quit      chan struct{}
 	wg        sync.WaitGroup
+
+	// log is a peer-specific logging instance.
+	log btclog.Logger
 }
 
 // A compile-time check to ensure that Brontide satisfies the lnpeer.Peer interface.
 var _ lnpeer.Peer = (*Brontide)(nil)
 
 // NewBrontide creates a new Brontide from a peer.Config struct.
 func NewBrontide(cfg Config) *Brontide {
+	logPrefix := fmt.Sprintf("Peer(%x):", cfg.PubKeyBytes)
```

Comment:
This needs to be added to `peer/brontide_test.go` as well, otherwise the tests run into a nil pointer panic.

Reply:
Fixed. In fact I fixed that before, but somehow this change got lost again. `git reflog` to the rescue :)